src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt

```diff
@@ -94,36 +66,76 @@ suspend fun Bot.handleParty(msg: Message, args: String?) {
         return
     }
 
-    var hasInvalidRes = false
-    val sb = StringBuilder()
-    parsedArgs.forEach {
-        if (it == "admins") {
-            sb.append(pullAdminsParty(msg) ?: "")
-            return@forEach
-        }
+    handleParty(parsedArgs.asSequence(), msg) {
+        sendMessage(
+            chatId,
+            if (parsedArgs.size == 1) ON_PARTY_REQUEST_FAIL
+            else ON_PARTY_REQUEST_FAILS
+        )
+    }
+}
+
+/**
+ * Handle implicit `@party-name`-like calls
+ */
+suspend fun Bot.handleImplicitParty(msg: Message) {
+    val text = msg.text
 
-        val res = partyCommandTransaction(chatId, it)
-        if (res.isNullOrBlank()) {
-            hasInvalidRes = true
-            return@forEach
+    if (msg.forward_from != null || text == null) {
+        return
+    }
+
+    val partyNames = text.lineSequence()
+        .flatMap { it.split(' ', '\t').asSequence() }
+        .filter { it.startsWith('@') }
+        .map { it.substring(1) }
```

Comment:
Consider using `removePrefix` here, instead of filter and map: it removes prefix only if it exists and makes this part more readable